cms/static/coffee/spec/views/course_info_spec.coffee

```diff
@@ -167,22 +167,23 @@ define ["js/views/course_info_handout", "js/views/course_info_update", "js/model
                 @courseInfoEdit.render()
                 @event = {preventDefault : () -> 'no op'}
                 @courseInfoEdit.onNew(@event)
-                @requests = AjaxHelpers["requests"](this)
```

Comment:
@nasthagiri FYI. We hadn't done a good job communicating the strange pattern that we have discovered needs to be followed for mocking in JS tests that have ajax requests in order for the tests to not be flaky. :(
Tagging you here so that you are aware of it, in case you see it come up in code reviews, etc.